chunk_arrow.go

```diff
@@ -18,7 +18,7 @@ type arrowResultChunk struct {
 }
 
 func (arc *arrowResultChunk) decodeArrowChunk(rowType []execResponseRowType) ([]chunkRowType, error) {
-	logger.Info("Arrow Decoder")
+	logger.Debug("Arrow Decoder")
```

Comment:
could this have the context too?

Reply:
Context is not yet available in arrowResultChunk struct. And entering an internal function is trivial. So I will leave this to future if someone feel they need this log.